### ISSUE TITLE
Uncaught ReferenceError: _interval is not defined when render_aync with Selector

### ISSUE THREAD

**Issue by fffx** [nikolalsvk/render_async#92]
I use render_async with: 

```
render_async some_path,  toggle: { selector: "#sync_load_trigger" }
```
when I click sync_load_trigger button I got 
```
(index):11015 Uncaught ReferenceError: _interval is not defined
    at HTMLLIElement.<anonymous> ((index):11015)
```

render_async version:  2.1.2

My workaround  is  define  a global variable `_interval`  before render_async
```
 <script type="text/javascript">
       _interval = null
 </script>
```

**Comment by nikolalsvk** [comment]
Fix for this is here https://github.com/renderedtext/render_async/pull/93

**Comment by nikolalsvk** [comment]
I've just released new version 2.1.3! Feel free to check if everything is OK https://rubygems.org/gems/render_async/versions/2.1.3
